### DISCUSSION TITLE
Welcome to Innovation-Challenge Discussions!

### DISCUSSION THREAD

**Discussion by MatthewCalder-msft** [msusdev/Innovation-Challenge#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the Innovation Challenge community
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mothicc** [comment]
Greetings, are we going to have azure accounts for our projects, or any kind of resources from microsoft? So excited to be part of this hackaton 💖💖

**Reply by MatthewCalder-msft** [reply]
I'm sorry but we are not able to provide Azure subscriptions for the hackathon. It is something we'd like to be able to do in the future. But you should be able to build a working proof of concept with the services that are free with the Azure free trial- the ones that are free for 12 months or always free. [You can find details if you scroll down on this page](https://azure.microsoft.com/en-us/free/free-account-faq/)
Excited to have you participating!

**Comment by ggrossvi** [comment]
Can you tell us more info about the relationship of Women in Cloud and LinkedIn.  Are they a partner with LinkedIn so we can identify which apis are available to us?  I believe that to request a key you need to create an app and link it to a company page.  Are we suppose to link it to Women in Cloud or to a demo company page for the hackathon?    Are there any groups that have gotten a LinkedIn  API key?  If you have gotten a key can you share what you did to get one.  Thanks!  Also are we limited to just one advisor or can approach whoever is available.   Is there a forum for us to ask questions to all of them.  Thanks!

**Reply by MatthewCalder-msft** [reply]
I don't know if Women in Cloud are in the LinkedIn partner programs or if any teams have been able to get access to the LinkeIn APIs. I think that the [Consumer API may be the only one that does not require you to be a partner to access](https://learn.microsoft.com/en-us/linkedin/shared/authentication/getting-access). When we put that challenge out there we were concerned the LinkedIn terms of use would be too restrictive, so the challenge does allow you to take a different approach other than LinkedIn APIs. You may decide you want to take on a different executive challenge, but if you want to stick with solving for that one, keep with the goal of helping people understand the effectiveness of their networks, but try something like [Gephi ](https://gephi.org/) [Developer's Handbook](https://github.com/gephi/gephi/wiki/Developer-Handbook) Focus on building a proof of concept that shows how your idea could work.

You can work with multiple advisors. I think there is a limitation on the hackbox platform that you can only join 1 project even if you are an advisor, but feel free to contact as many advisors as you find helpful. (You just may need to communicate mostly by email)

**Comment by 90kristin** [comment]
Hi, My name is Kristin and I am currently almost done completing Microsoft Cybersecurity Analyst course with Women In Cloud. I am very excited to be here and participating in the June Innovation Challenge. I will like to know how do I go about getting assigned to a team? I have put in a request to be added to a project but I am still waiting on a response from the project owner. Any assistance would be greatly appreciated. Thanks!

**Reply by MatthewCalder-msft** [reply]
If there is a project you want to join, feel free to also send email to the project owner or to reach out to them on LinkedIn. You should be able to find both their email and their LinkedIn on their profile within the Innovation Studio. If the project owner isn't getting back to you, you should also keep looking for another project to join or start your own and invite a couple people. I'm also keeping a list of people looking to join a team. Feel free to send mail to me directly at macalde@microsoft.com

**Reply by 90kristin** [reply]
Ok, thank you and will do!

**Comment by 73LidiaBenitez** [comment]
Hi, One question, when will the list of winners be released? I've seen some extraordinary projects.

**Comment by MatthewCalder-msft** [comment]
The judges are wrapping up today and I'll be posting the winners tomorrow. Some extraordinary projects for sure!

**Reply by 73LidiaBenitez** [reply]
Hello Matthew, do you have the list of winners, or where we can find it? I really appreciate your attention. Best regards.

**Reply by MatthewCalder-msft** [reply]
You can find t[he recent winners here](https://techcommunity.microsoft.com/discussions/azure/announcing-the-winners-of-the-june-innovation-challenge/4427978)  as well as on the home page of the hackathon site
